core/color.h

```diff
@@ -187,6 +188,7 @@ struct Color {
 	static Color hex(uint32_t p_hex);
 	static Color html(const String &p_color);
 	static bool html_is_valid(const String &p_color);
+	static const Map<String, Color> &names();
```

Comment:
I would simply not add this here, its not going to be used anyway, just put them together with the other constants inside Variant

Reply:
Done.